docs/book/helpers/html-tag.md

```diff
@@ -1,3 +1,8 @@
+<!-- markdownlint-configure-file {
+    "no-trailing-spaces": false,
+    "code-block-style": false
```

Comment:
The rule should not disable for the entire document, only temporarily where the problems arise.